src/pixi-multistyle-text.ts

```diff
@@ -879,7 +879,9 @@ export default class MultiStyleText extends PIXI.Text {
 
 		let dropShadowPadding = this.getDropShadowPadding();
 
-    texture.baseTexture.setRealSize(this.canvas.width, this.canvas.height, this.resolution);
+    if (texture.baseTexture.setRealSize) {
```

Comment:
This exists in Pixi v5, which is the only version we're now supporting going forward.
If you need this, it's because you're still using Pixi v4, therefore you should use pixi-multistyle-text v0.8.0

Reply:
If this is the case then we should just message the user if the version is old. I added another commit. 

Also, the reason this is happening in the demo is because the HTML for the demo page is loading the old version of Pixi.
`<script src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/4.5.6/pixi.js"></script>`